apps/web/src/pages/ListsPage.tsx

```
@@ -69,34 +69,34 @@ function AddItemModal({
   };
 
   return (
-    <div className="fixed inset-0 z-50 flex items-start justify-center bg-black/60 backdrop-blur-sm pt-[10vh]" onClick={onClose}>
+    <div className="fixed inset-0 z-50 flex items-start justify-center bg-black/70 backdrop-blur-sm pt-[10vh]" onClick={onClose}>
```

Biome (2.4.6)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)